docs/guides/lint.md

```diff
@@ -3,75 +3,235 @@ redirectFrom:
+      severity: error
+    boolean-parameter-prefixes:
+      severity: error
+      prefixes: ['should', 'is', 'has']
+  ...
+```
+
+## Sub-sections
+
+### Plugins
+
+Use this section to import local plugins.
 
-When using our hosted Workflows product, the `.redocly.yaml` file must be in the root of the repository.
+* **type**: `array of strings`
 
+:::warning
+You don't need to import built-in plugins and rules.
+
+Community plugins are not supported.
 :::
 
-Read about the [`--config` option](../commands/index.md) to use other file names or locations.
+:::success
+Omit this section if you don't have custom plugins.
+:::
 
-From a high-level, there are a few sub-sections.
+#### Examples
 
-```yaml
+```yaml single value
 lint:
   plugins:
-    # This section is where you can import local plugins.
-    # We don't support community plugins.
-    # You don't need to import our built-in plugins and rules.
-    # Omit this section if you don't have custom plugins.
     - './local-plugin.js'
+```
+
+```yaml multiple values
+lint:
+  plugins:
+    - ['./local-plugin.js', './another-local-plugin.js']
+```
+
+### Extends
 
+Use this section to choose the base configuration for further extension or adding your own plugins. You may override specific settings in the subsequent sections.
+
+* **type**: `array of strings`
+* **default**: `recommended`
+* **possible values**: `minimal`, `recommended`, `all`
+
+:::info
+Possible values specified above are related to built-in rules only. Custom plugins can contain additional configurations, so it can be as follows:
+
+```yaml
+lint:
+  extends:
+    - recommended
+    - my-custom-super-ruleset
+```
+
+Find more information in the [Configs in plugins](../resources/custom-rules/#configs-in-plugins) section.
+
+:::
+
+#### Examples
+
+```yaml single value
+lint:
   extends:
-    # This section is where you choose the base configurations.
-    # You may override specific settings in the subsequent sections.
-    - recommended # This is the default (and built in) configuration. If it is too strict, try `minimal`.
+    - minimal
+```
+
+```yaml multiple values
+lint:
+  extends:
+    - minimal
+    - recommended
+```
+
+### Resolve
+
+Use this section to specify external links in your definition that are not publicly accessible (except for Redocly API registry links).
+
+* **type**: `object`
+
+Redocly automatically resolves any API registry link or public URL. However, you may want to resolve links that are not API registry links or publicly accessible.
+
+Currently, OpenAPI CLI only supports `http` headers. Only one `http` header per URL is supported in the `resolve` section:
+
+```yaml
+lint:
+  resolve:
+    http:
+      headers:
+        - # header configuration
+```
+
+#### Header configuration
+
+| Property      | Description | Examples |
+| ------------- | ----------- | -------- |
+| `matches`     | Glob match against the URL. | `https://api.example.com/**` or `https://api.example.com/*.json` |
```

Comment:
query: is "glob" short for "global"? I would use **global** in that case, unless glob is really the word you were going for.

Reply:
Glob is the right word here. It is a pattern that matches files or directories or other pathnames based on wildcards (*, **).